glue/sales_val_flagging.py

```diff
@@ -427,6 +427,17 @@ def finish_flags(df, start_date):
     )
     run_id = timestamp + "-" + random_words
 
+    # Control flow for incorporating versioning
+    dynamic_assignment = {
+        "run_id": run_id,
+        "rolling_window": lambda df: pd.to_datetime(
+            df["rolling_window"], format="%Y%m"
+        ).dt.date,
+    }
+
+    if not manual_update:
+        dynamic_assignment["version"] = 1
+
```

Comment:
I had removed this because I thought it was redundant but I was mistaken. This is here to manually assign version of 1 when `manual_update` is false.